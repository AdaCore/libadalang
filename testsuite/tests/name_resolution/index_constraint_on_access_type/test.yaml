driver: name-resolution
input_sources: [a.adb]
