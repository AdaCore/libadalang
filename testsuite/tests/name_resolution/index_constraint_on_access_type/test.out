Analyzing a.adb
###############

Resolving xrefs for node <ObjectDecl ["X"] a.adb:5:4-5:28>
**********************************************************

Expr: <Id "Arr_Access" a.adb:5:8-5:18>
  references:    <DefiningName "Arr_Access" a.adb:3:9-3:19>
  type:          None
  expected type: None
Expr: <BinOp a.adb:5:20-5:26>
  type:          <ConcreteTypeDecl ["Integer"] __standard:4:3-4:54>
  expected type: <ConcreteTypeDecl ["Integer"] __standard:4:3-4:54>
Expr: <Int a.adb:5:20-5:21>
  references:    None
  type:          <ConcreteTypeDecl ["Universal_Int_Type_"] __standard:116:3-116:45>
  expected type: <ConcreteTypeDecl ["Integer"] __standard:4:3-4:54>
Expr: <OpDoubleDot ".." a.adb:5:22-5:24>
  references:    None
  type:          None
  expected type: None
Expr: <Int a.adb:5:25-5:26>
  references:    None
  type:          <ConcreteTypeDecl ["Universal_Int_Type_"] __standard:116:3-116:45>
  expected type: <ConcreteTypeDecl ["Integer"] __standard:4:3-4:54>


Done.
